server-spi/src/main/java/org/keycloak/utils/StringUtil.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.keycloak.utils;
+
+public class StringUtil {
+
+    public static boolean isBlank(String str) {
+        return !(isNotBlank(str));
+    }
+
+    public static boolean  isNotBlank(String str) {
+        return str != null && !str.isEmpty();
```

Comment:
Trim the string before checking if it is empty?

Reply:
@pedroigor Yes, I will fix codes as https://github.com/keycloak/keycloak/blob/master/saml-core/src/main/java/org/keycloak/saml/common/util/StringUtil.java .